### ISSUE TITLE
Add Frechet Inception Distance (FID) metric

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#62]

**Comment by jshin1394** [comment]
Using torchmetric FID [implementation](https://lightning.ai/docs/torchmetrics/stable/image/frechet_inception_distance.html) as base, add the metric to metrax.

**Comment by jshin1394** [comment]
deprioritizing as it requires a pre-trained InceptionV3 model
